apps/website/package.json

```diff
@@ -18,6 +18,7 @@
     "@fontsource/noto-sans": "^4.5.11",
     "@fontsource/noto-sans-mono": "^4.5.11",
     "@itwin/itwinui-react": "*",
+    "@itwin/itwinui-icons-react": "*",
```

Comment:
is this needed?

Reply:
This was an attempt to solve this error:
![image](https://user-images.githubusercontent.com/83585998/210390975-bf812f6e-f7c7-4b04-9980-5e6edfd6c09d.png)

Did not succeed.

Reply:
oh that one is inside the sandbox, so changing package.json will not help.

try adding it to [`LiveExample.tsx`](https://github.com/iTwin/iTwinUI/blob/main/apps/website/src/components/LiveExample.tsx#L88)

Reply:
Can you use `latest` for both packages?

Reply:
Updated